### DISCUSSION TITLE
Is there any documentation for @deepkit/type?

### DISCUSSION THREAD

**Discussion by cakoose** [marcj/deepkit#102] [Q&A] [answered]
I'm trying to pick a library for TypeScript-aware serialization/deserialization.  Is there any documentation for @deepkit/type?

**Comment by marcj** [comment, marked as answer]
Yes, we have a beta build of the website, which is nowhere linked (because it's nowhere complete). You can use it as long as we are not fully released yet.
https://beta.deepkit.io/documentation/type
and product page https://beta.deepkit.io/library/type

**Reply by marcj** [reply]
Website is now available at https://deepkit.io/documentation/type. Not quite done, but enough for most people.
